### ISSUE TITLE
Transmission 2.84-10 on DS214 stopped working after updating to DSM 5.1-5022 Update 4

### ISSUE THREAD

**Issue by copyl3ft** [SynoCommunity/spksrc#1619]
Hello,
ive been using your package of Transmission 2.84-10 on DS214. It was flawless until DSM 5.1-5022 Update 4 came. After I updated to it, my Transmission stopped working, literally. So I manually restarted it through package center but it wont restart. I contacted support center but they dont have answer yet.

The impatient me cant find an answer anywhere so I decided to uninstall the package and re-installed it. It got installed and run, unfortunately, it will stop working just moments after transmission starts downloading a torrent. When I click the log from Package Center, nothing is there so I dont know where to find more information to what is happening to the package.

**Comment by Dr-Bean** [comment]
Have you checked that the `users` group has sufficient access to the folders you use for Transmission? The permissions should be set to read/write on the shared folders and subfolders that you use. You can do that via the DSM interface.

With that confirmed, what happens when you enter this on a command line (Putty/Terminal) ?
```su - transmission -c "PATH=/usr/local/transmission/bin:/usr/local/python/bin:${PATH} /usr/local/transmission/bin/transmission-daemon -g /usr/local/transmission/var/ -x /usr/local/transmission/var/transmission.pid"```

**Comment by copyl3ft** [comment]
Hi,
users group has sufficient access:
![trans2](https://cloud.githubusercontent.com/assets/9149535/6910638/54b73f46-d788-11e4-9b86-af645bd12846.png)


after typing the command in Putty this is what happens:
![trans](https://cloud.githubusercontent.com/assets/9149535/6910649/71edcf94-d788-11e4-9cfe-e0f681e37996.png)


and this will happen when I add a torrent:
https://www.youtube.com/watch?v=Bk21B8gTilI

**Comment by Dr-Bean** [comment]
If Transmission starts without errors, then it seems more likely to be a permissions issue (which it encounters once it starts downloading files) I don't know what configuration you have, but maybe recheck that.

As for the video, it looks like you use a different interface then the web interface provided with the package (which runs on port 9091). The web interface has an 'information' button, which should show the error that occurs on that particular torrent. I assume your client has the same ability, if not, use the web interface, and check what it says.

**Comment by copyl3ft** [comment]
FileCenterV2# cat /volume1/@appstore/transmission/var/settings.json
{
    "alt-speed-down": 50,
    "alt-speed-enabled": false,
    "alt-speed-time-begin": 540,
    "alt-speed-time-day": 127,
    "alt-speed-time-enabled": false,
    "alt-speed-time-end": 1020,
    "alt-speed-up": 50,
    "bind-address-ipv4": "0.0.0.0",
    "bind-address-ipv6": "::",
    "blocklist-enabled": false,
    "blocklist-url": "http://www.example.com/blocklist",
    "cache-size-mb": 4,
    "dht-enabled": true,
    "download-dir": "/volume1/Downloads/DLStation",
    "download-queue-enabled": true,
    "download-queue-size": 5,
    "encryption": 2,
    "idle-seeding-limit": 30,
    "idle-seeding-limit-enabled": false,
    "incomplete-dir": "/volume1/Downloads/incomplete",
    "incomplete-dir-enabled": true,
    "lpd-enabled": false,
    "message-level": 2,
    "peer-congestion-algorithm": "",
    "peer-id-ttl-hours": 6,
    "peer-limit-global": 200,
    "peer-limit-per-torrent": 50,
    "peer-port": 51413,
    "peer-port-random-high": 65535,
    "peer-port-random-low": 49152,
    "peer-port-random-on-start": false,
    "peer-socket-tos": "default",
    "pex-enabled": true,
    "pidfile": "/usr/local/transmission/var/transmission.pid",
    "port-forwarding-enabled": true,
    "preallocation": 1,
    "prefetch-enabled": 1,
    "queue-stalled-enabled": true,
    "queue-stalled-minutes": 30,
    "ratio-limit": 2,
    "ratio-limit-enabled": false,
    "rename-partial-files": true,
    "rpc-authentication-required": false,
    "rpc-bind-address": "0.0.0.0",
    "rpc-enabled": true,
    "rpc-password": "{dc29b8c1dd78b688b0ffdaab85b7760c3bfe0395KtMh6XLG",
    "rpc-port": 9091,
    "rpc-url": "/transmission/",
    "rpc-username": "admin",
    "rpc-whitelist": "127.0.0.1",
    "rpc-whitelist-enabled": false,
    "scrape-paused-torrents-enabled": true,
    "script-torrent-done-enabled": false,
    "script-torrent-done-filename": "",
    "seed-queue-enabled": false,
    "seed-queue-size": 10,
    "speed-limit-down": 200,
    "speed-limit-down-enabled": true,
    "speed-limit-up": 32,
    "speed-limit-up-enabled": true,
    "start-added-torrents": true,
    "trash-original-torrent-files": false,
    "umask": 2,
    "upload-slots-per-torrent": 14,
    "utp-enabled": true,
    "watch-dir-enabled": false
}
FileCenterV2#

**Comment by copyl3ft** [comment]
What am I missing?

**Comment by Dr-Bean** [comment]
Well, great that you solved it :) 

On a more serious note: Please people, don't ask for support here when you have ever in your life used that 'tutorial'...because it doesn't prevent headaches what-so-ever...

**Comment by copyl3ft** [comment]
Im sorry if I did asked for help when i used other tutorial with your package because I forgot doing so when I installed Transmission. It was working fine for almost a year then after the latest update came and it just stopped working. I thought something in that update is conflicting with the Transmission package you guys published thats why I resorted to asking for help here. I like Transmission and Im so grateful that you guys have a working package for Synology because for me, Download Station is not even close to this package. Dont worry next Ill be more careful in classifying the problem
